### ISSUE TITLE
Constructors and record fields are indexed under an unconventional name

### ISSUE THREAD

**Issue by art-w** [art-w/sherlodoc#12]
For example, it's not possible to search for `Seq.Cons` or `Stdlib.contents` as they are indexed under the invalid name `Seq.node.Cons` and `Stdlib.ref.contents` :

https://doc.sherlocode.com/?q=Seq%20Cons
https://doc.sherlocode.com/?q=Stdlib%20contents

**Comment by EmileTrotignon** [comment]
This seems to be fixed by https://github.com/art-w/sherlodoc/pull/23 on the online version, but not in master, your link does not show the issue anymore.

**Comment by art-w** [comment]
Yes the search works a lot better with #23 but the names presented to users are still unconventional / invalid OCaml syntax :)
